### ISSUE TITLE
Make TT_Eval() function that evaluates an SQL query

### ISSUE THREAD

**Issue by edwardsmarc** [edwardsmarc/PostgreSQL-Table-Translation-Framework#205]
Function should take a stringlist of the values or columns to be used, and an sql query string to evaluate.
Will then need to find a way to insert the values into the sql query.

How can the values be linked to the query?
* Maybe the sql query uses a, b, c etc as placeholders for the values. The function can then replace a, b, c with the values from the stringlist in the order they appear.

**Comment by pedrogit** [comment]
The eval() rule should be written like other rules except that it take only a string and all column names and stringlist in the string would be automatically replaced by source values. e.g.:

eval(col1 - col2)

**Comment by edwardsmarc** [comment]
@pedrogit I guess we can use the existing parsing functions in the engine  to do this?

**Comment by pedrogit** [comment]
No. We'll need a new parsing algo. The one we have is able to parse a comma separated list of argument. Not an arbitrary string containing attribute names. I'll see what we can do...
